docs/site/Controllers.md

```diff
@@ -10,83 +10,109 @@ summary:
 
 ## Overview
 
-A `Controller` is a class that implements operations defined by application's REST API. It implements an application's business logic and acts as a bridge between the HTTP/REST API and domain/database models.
-A `Controller` operates only on processed input and abstractions of backend services / databases.
+A `Controller` is a class that implements operations defined by application's
+REST API. It implements an application's business logic and acts as a bridge
+between the HTTP/REST API and domain/database models. A `Controller` operates
+only on processed input and abstractions of backend services / databases.
 
 ### Review questions
```

Comment:
What's the purpose of this section?

Reply:
I think this section is to remind whoever that was supposed to finish this page what to cover. I'm going to be opening up an issue to replace the content in this page (along with a couple of others) that show examples of bottom-up approach to configure routing (`@get` and what not).